File: students/students/src/MON.SPA/src/models/diploma/barcodeYearModel.js
export class BarcodeYearModel {
    constructor(obj = {}) {
      this.id = obj.id || 0;
      this.edition = obj.edition || null;
      this.schoolYear = obj.schoolYear || null;
      this.headerPage = obj.headerPage || null;
      this.internalPage = obj.internalPage || null;
      this.basicDocumentId = obj.basicDocumentId || null;
    }
  }

File: students/students/src/MON.Models/Diploma/BarcodeYearModel.cs
using System.Collections.Generic;

namespace MON.Models.Diploma
{
    public class BarcodeYearModel
    {
        public int Id { get; set; }
        public short Edition { get; set; }
        public short SchoolYear { get; set; }
        public string HeaderPage { get; set; }
        public string InternalPage { get; set; }
        public int BasicDocumentId { get; set; }
    }

    public class BarcodeYearListViewModel
    {
        public string DiplomaTypeName { get; set; }
        public IEnumerable<BarcodeYearModel> BarcodeYears { get; set; }
    }
}
